code/modules/mob/living/carbon/human/species_types/modder.dm

```diff
@@ -0,0 +1,98 @@
+//gene modded humans
```

Comment:
If you're wanting to replace Felinids with Modders in the character creation species list, [`game_options.txt`](https://github.com/pariahstation/Pariah-Station/blob/b7477dad8f9384cfc9e37c88b570e158b1ec097d/config/game_options.txt#L307) will need editing too.
Something like this would work:
```diff
## Races that are strictly worse than humans that could probably be turned on without balance concerns
-ROUNDSTART_RACES felinid
+#ROUNDSTART_RACES felinid
+ROUNDSTART_RACES modder
ROUNDSTART_RACES lizard
```

Reply:
i think thats something the host has to do on their end

Reply:
It is, but it'd be good to have them on people's test servers as well.